community/kernel/src/main/java/org/neo4j/graphdb/impl/notification/NotificationCode.java

```diff
@@ -184,7 +184,12 @@
     START_UNAVAILABLE_FALLBACK(
             SeverityLevel.WARNING,
             Status.Statement.PlannerUnavailableWarning,
-            "START is unsupported for current CYPHER version, the query has been execute by an older CYPHER version"
+            "START is not supported for current CYPHER version, the query has been executed by an older CYPHER version"
```

Comment:
It is still around, just changed the wording. So my thinking was that it is two different notifications, one is saying: _this query could not be run on this version of cypher_ and the other one is saying: _`START` is deprecated`.  It is really two different things, or is my thinking wrong?